frontend/src/components/House/HouseList.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import House from './House';
+import {FilteredIssueList} from '../IssueList';
+import {css} from 'aphrodite';
+import S from './styles';
+
+export default class HouseList extends React.Component {
+  render = () => {
+    const {data} = this.props;
+    const house_list = data.map( (house, ndx) => {
```

Comment:
I believe you should be able to get this shortened into one render statement. If you wrap the whole map where you've put {house_list} then you won't need the other return. Map should auto return this....

`return (Objects.map(x => <div>{x.value}</div>));` should work... I find this just looks a bit messy.

Reply:
`map` is a method on `Array` like objects. IMHO I find this approach better because it separates logic from presentation.